### ISSUE TITLE
exception in GotoLineCol.dll Notepad++ v8.3.3   (32-bit)

### ISSUE THREAD

**Issue by hopalongrock** [shriprem/Goto-Line-Col-NPP-Plugin#22]
I didn't use the plugin explicitly.

Notepad++ v8.3.3   (32-bit)
Build time : Mar 13 2022 - 17:11:10
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Command Line : D:\00ROCKABILLY.txt
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Home (64-bit) 
OS Version : 2009
OS Build : 19044.1620
Current ANSI codepage : 1250
Plugins : AutoCodepage.dll combine.dll ComparePlugin.dll DSpellCheck.dll GotoLineCol.dll LocationNavigate.dll LuaScript.dll mimeTools.dll NppBookmarkManager.dll NppCalc.dll NppConverter.dll NppExport.dll NPPJSONViewer.dll NppSaveAsAdmin.dll NppTextFX.dll PluginManager.dll 

![20220412 073138  _D__00ROCKABILLY txt - Notepad++](https://user-images.githubusercontent.com/6840865/162902962-acbfe799-1841-411c-bff7-730999b11909.jpg)

![20220412 073500  _D__00ROCKABILLY txt - Notepad++](https://user-images.githubusercontent.com/6840865/162902999-90789dce-4126-42c6-8750-c0eccea64b57.jpg)

**Comment by shriprem** [comment]
@hopalongrock, I am unable to replicate the issue while using the exact command line parameters that you pasted in the issue report.

The exception you received occurred while the command line parameters were being processed.

1. Did the exception happen when the full command line instruction was exactly `C:\Program Files (x86)\Notepad++\notepad++.exe D:\00ROCKABILLY.txt`? Or was it something different? If so, please provide me the different instruction that was used in its **_full, exact_** version.

2. Were you trying to use the GLC command line parameters?

From the online document on the GLC [Command Line Options](https://github.com/shriprem/Goto-Line-Col-NPP-Plugin/blob/master/docs/CommandLineOptions.md):

>#### Sample Usage with all options specified:
```
NPP_DIR\notepad++.exe DirPathA_File1 DirPathA_File2 DirPathB_File3 -n8 -c42 -z -GLCb1;c0;d1;e18;f1;h1;p1;q1
```

If you make a mistake in specifying the GLC command line options, like using for example `-GLCbz` wherein you provided a non-numeric character like 'z', the current version of the plugin will spit out the exception message that you saw. If so, please fix that.  Refer to the GLC [Command Line Options](https://github.com/shriprem/Goto-Line-Col-NPP-Plugin/blob/master/docs/CommandLineOptions.md) document.

**Comment by hopalongrock** [comment]
I'm sorry, I made the NPP debug information a little bit later, maybe I was edited not only one file, when the issue appeared - so I don't know exactly the file name.
The error didn't cause any problem, I could continue normally, what I wanted to do.
I don't use GLC, I start NPP from another program (FCW File Commander) with the Start command, only with the filename parameter.
I use NPP this way from years, only the program versions change.

This occurred only one time, I will be watching to see if it happens again.

Thank you for your help.

**Comment by hopalongrock** [comment]
Now I have NPP 8.4, but the plugin version didn't change (2.2.0.2, 32 bit), and there's no plugin update available. On https://github.com/shriprem/Goto-Line-Col-NPP-Plugin
the last version is also 2.2.0.2.

I have this exception again, I just started NPP and tried to resize its window. The resize was unsuccessful, no problem with a second resize.

**Comment by hopalongrock** [comment]
Thanks, I copied the dll into my GotoLineCol plugin folder,

**Comment by shriprem** [comment]
I was finally able to replicate the "Exception Reason: invalid stoi argument" error message with the **_older 2.2.0.2_** version of the GotoLineCol plugin: by trying to launch Notepad++ with a file whose name either started with '-n' or '-c' as in file names: "-newfile.bat" & "-changefile.txt". 

File extensions didn't matter. Only the file name has to  start with either a '-c' or '-n'. See the sample clip below:

![image](https://user-images.githubusercontent.com/45252729/165885271-658902d9-5719-4f2e-b117-b317559cb2f3.png)

I can also confirm that this error message did not occur when I tried the same files with the [2.2.0.3 special versions](https://github.com/shriprem/Goto-Line-Col-NPP-Plugin/issues/22#issuecomment-1112324114) of the GotoLineCol plugin I had provided earlier yesterday.

If these file name patterns ring a bell, please let me know so that I can put this issue to rest. Thanks!

**Comment by hopalongrock** [comment]
It's great.
But I never start a file with the "-" character (I have no such file), and in the last case I started  NPP without any parameter (no filename, no option, nothing) - however I didn't save the exception text, so maybe the "stoi argumen"t didn't appear.
I'm watching what happens using the new dll.

**Comment by shriprem** [comment]
Fixed with Release [2.3.0.0](https://github.com/shriprem/Goto-Line-Col-NPP-Plugin/releases/tag/v2.3.0.0). Closing the issue.
